web/src/infra/gameInfo/GameContributors.tsx

```diff
@@ -0,0 +1,35 @@
+import { IGameDef } from 'gamesShared/definitions/game';
+import React from 'react';
+import { Typography } from '@material-ui/core';
+
+interface GameContributorsProps {
+  game: IGameDef;
+}
+
+
+export class GameContributors extends React.Component<GameContributorsProps, {}> {
```

Comment:
It doesn't really make any difference (so you can ignore it for this PR if you don't want to bother), but this seems like a good candidate for a functional component.

```typescript
export default function GameContributors({game}: GameContributorsProps) {
    // ...
    return <p>foo</p>;
}

Reply:
Done.